Test failed!
Test failed!
Test failed!
Test failed! Error: HTTPConnectionPool(host='127.0.0.1', port=5000): Max retries exceeded with url: /score (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000028775EA0B50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
Test failed!
Test failed! Error: 404 Client Error: NOT FOUND for url: http://127.0.0.1:5000/score
Test failed!
Test failed! Error: HTTPConnectionPool(host='127.0.0.1', port=5000): Max retries exceeded with url: /score (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001766A7B2E50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
Test failed!
Test failed! Error: 404 Client Error: NOT FOUND for url: http://127.0.0.1:5000/score
Test failed!
Test failed! Error: HTTPConnectionPool(host='127.0.0.1', port=5000): Max retries exceeded with url: /score (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001C63D5986D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
Test failed!
Test failed! Error: HTTPConnectionPool(host='127.0.0.1', port=5000): Max retries exceeded with url: /score (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001EF324CE990>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
Test failed!
Test failed! Error: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
Test passed!
Test failed! Error: HTTPConnectionPool(host='127.0.0.1', port=5000): Max retries exceeded with url: /score (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001A9187A24D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
Test passed!
Test failed! Error: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
Test failed! Error: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
Test failed! Error: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
Test failed! Error: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
